rs/ethereum/ledger-suite-orchestrator/src/candid/mod.rs

```diff
@@ -25,6 +25,7 @@ pub struct UpgradeArg {
     pub index_compressed_wasm_hash: Option<String>,
     pub archive_compressed_wasm_hash: Option<String>,
     pub cycles_management: Option<UpdateCyclesManagement>,
+    pub manage_other_canisters: Option<Vec<ManageOtherCanisters>>,
```

Comment:
>  "manage other canisters" could be misinterpreted as "now manage these canisters (and forget about what you did so far)".

I didn't think this could be a source of confusion (but I see your point).

> Did you consider making this more explicit, for example add_ledger_suite: Option<Vec<LedgerSuite>> or something along these lines?

I thought about something with `add_`; however, I thought that this would be misleading since there is already `AddErc20Arg`, but this last one does create new canisters, while the currently named `manage_other_canisters` does not. My other line of thinking was to name it something like `manage_already_installed_canisters`, WDYT?

Reply:
Actually, the word `Canisters` bothers me more than `manage`.   :slightly_smiling_face:

I see your point regarding `add_`. How about `manage_existing_ledger_suite` (if I may suggest a replacement for `Canisters` again)?